crates/mysten-metrics/src/lib.rs

```diff
@@ -49,6 +51,13 @@ impl Metrics {
                 registry,
             )
             .unwrap(),
+            channels: register_int_gauge_vec_with_registry!(
+                "monitored_channels",
+                "Size of channels per callsite.",
+                &["callsite"],
```

Comment:
Maybe call the label `name`? Other `callsite` labels have actual `file:line` as values.

Reply:
I thought about it and thought it would be more consistent if we stick with one label for these `monitored` metrics

Reply:
monitored_scope is using 'name' as label name. We may migrate monitored tasks and futures too.

Reply:
you are right, I will use `name` for this